module/sell.py

```diff
@@ -1,20 +1,33 @@
+import bs4
+import re
 from telegram import Update
 from telegram.ext import CallbackContext
-import bs4
 from module.add_item import add_item
 from module.add_book import add_book
 from module.find import find
 from module.book_in_unict import book_in_unict
 from module.create_connection import create_connection
+from module.send_results import get_book_info
 from module.shared import DB_PATH, error_message
 
 
 def _get_isbn_from_website(soup: bs4.BeautifulSoup) -> str:
     idx = len(str(soup.findAll("td")).split("bibInfoData")) - 1
-    return str(soup.findAll("td")) \
+    isbn = str(soup.findAll("td")) \
         .split("bibInfoData")[idx] \
         .split("\n")[1] \
         .split("<")[0]
+    isbn = isbn.replace('-','')
+    if len(isbn) > 13:
+        p1 = re.search('978', isbn)
+        p2 = re.search('979', isbn)
+        if p1 is not None:
+            s = p1.span()[0]
+            isbn = isbn[s:s+13]
+        elif p2 is not None:
+            s = p2.span()[0]
+            isbn = isbn[s:s+13]
+    return isbn
```

Comment:
you like to use `if` and `elif` where it is not efficient to use 😁 
```suggestion
    isbn = isbn.replace('-','')
    if len(isbn) > 13:
        p1 = re.search('978', isbn)
        if p1 is not None:
            s = p1.span()[0]
            return isbn[s:s+13]
        
        p2 = re.search('979', isbn)
        if p2 is not None:
            s = p2.span()[0]
            return isbn[s:s+13]

    return isbn
```